### ISSUE TITLE
[4.7.*] BC break - Palette exceptions got moved to new namespace

### ISSUE THREAD

**Issue by dmolineus** [contao/contao#378]
**Affected version(s)**

Contao 4.7.*

**Description**  

Since 68b0919cdd8be37a1953f18d40a57abe21429e40 the palette related exceptions to removed from namespace `Contao\CoreBundle\Exception`. This is a huge BC break because the class now doesn't exist anymore. Furthermore catch blocks might fail because exception is not caught anymore.

For backward compatibility reasons the new classes should extend from `Contao\CoreBundle\Exception\PaletteNotFoundException` and `Contao\CoreBundle\Exception\PalettePositionException` and not vice versa.

**How to reproduce**  

```php
try {
    PaletteManipulator::doSomethingThatThrowsAnException();
} catch (Contao\CoreBundle\Exception\PaletteNotFoundException $e) {
   echo 'I should be displayed but aren\'t since Contao 4.7.*.'
}
```

**Comment by leofeyer** [comment]
Seems we got the inheritance wrong. The new exception needs to inherit from the old one of course.

**Comment by leofeyer** [comment]
Fixed in e2944c7cf105430508df4c1965ea03997be05b98.
